app/ui.js

```diff
@@ -1452,20 +1447,14 @@ var UI = {
             // selectionStart is undefined in Google Chrome
             newLen = newValue.length;
         }
-        var oldLen = oldValue.length;
+        const oldLen = oldValue.length;
 
-        var backspaces;
-        var inputs = newLen - oldLen;
-        if (inputs < 0) {
-            backspaces = -inputs;
-        } else {
-            backspaces = 0;
-        }
+        let inputs = inputs;
+        let backspaces = inputs ? -inputs : 0;
```

Comment:
Does this even execute at all?

Reply:
Alright, I'm going to stop reviewing here. You need to check the existing feedback we've given you and fix those things.